### DISCUSSION TITLE
Could not find code for updating a dashboard setting

### DISCUSSION THREAD

**Discussion by useEffects** [grafana/grafana#66821] [Dashboards, Panels & Visualizations]
I want to limit the number of characters in each input fields when requesting for updating the dashboard. I have already changed that in frontend and now i want to work for it in backend too, I have tried my best to find the code which deals with updating the dashboard in backend. If possible please show me the code for it Thank you

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
